### PR TITLE
[IDB IGNORE] [MDB IGNORE] the great resprite

### PR DESCRIPTION
**Pull request by retlaw34**
## About The Pull Request

AKA: The great respriting
One last try at bungalow
https://github.com/OracleStation/OracleStation/pull/969/files
https://github.com/OracleStation/OracleStation
https://github.com/shiptest-ss13/Shiptest/pull/713
https://github.com/shiptest-ss13/Shiptest/pull/532
https://github.com/shiptest-ss13/Shiptest/pull/484
https://github.com/shiptest-ss13/Shiptest/pull/395
https://github.com/shiptest-ss13/Shiptest/pull/57
https://github.com/shiptest-ss13/Shiptest/
https://github.com/Whitesands13/Whitesands/pull/533
https://github.com/tgstation/tgstation/pull/61703

## Why It's Good For The Game

Use your god damn eyes

## Changelog
:cl:
add: some sprites
/:cl:

### CODE REVIEW

**Comment by bruhlookatthisdood** [comment]
why is it IDB ignore?
How will i see the icons?!!11!1312131

File: ModularBungalow/Secure_lockers.dm
Comment (Kitsunemitsu) [inline ModularBungalow/Secure_lockers.dm:17]: File this under armor/captain.
Almost all the captain armor is filed under here, except for the original (which will be moved)

**Comment by bruhlookatthisdood** [comment]
> How are we supposed to know if this resprite is good if you've put IDB ignore tag?

thats what im sayin

**Comment by bruhlookatthisdood** [comment]
im just gonna....

**Comment by retlaw34** [comment]
> why is it IDB ignore? How will i see the icons?!!11!1312131

The IDB bot might shit itself with how many icons were changed

**Comment by thevandie** [comment]
can you at least just take screenshots and upload them? It shouldn't be that hard to save all the preview images and post them here.

**Comment by retlaw34** [comment]
> can you at least just take screenshots and upload them? It shouldn't be that hard to save all the preview images and post them here.

No, mostly since the bot shit itself as predicted

File: ModularBungalow/clothing/neocoat.dm
Comment (Kitsunemitsu) [inline ModularBungalow/clothing/neocoat.dm:86]: Voted against by the community.

File: code/game/machinery/suit_storage_unit.dm
Comment (Kitsunemitsu) [inline code/game/machinery/suit_storage_unit.dm:54]: Voted against by the community

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]
While I appreciate your enthusiasm, Community votes cannot be overruled by developers without host, head admin, or head coder approval.

As an aside, People have been running around and breaking the new SSUs because they do not like the look.
The labcoats in particular were voted as follows: 
1 For keeping the original
1 For swapping to the new ones
6 for keeping both.

I just am putting this out there before the live test. Rest is good to go.
We are a community-focused server, First and foremost. If you have an issue, bring it up with everyone.
If you have issues, you can bring them up with Xan, Hayden or I to try to push them through.

File: ModularBungalow/clothing/neocoat.dm
Comment (retlaw34) [reply ModularBungalow/clothing/neocoat.dm:86]: The point of the longcoat resprite was supposed to replace the old ones, not add an alternative. If youre just going to keep the old one anyways, then what's the point of a resprite?

Comment (thevandie) [reply ModularBungalow/clothing/neocoat.dm:86]: V a r i e t y

plus, it lets people choose whatever they want. This is why so many people hate combat mode, and one of the reasons we get new players. We try not to force new things down your throat and let you choose what you prefer. This carries over to fashion.

Comment (retlaw34) [reply ModularBungalow/clothing/neocoat.dm:86]: Its resprite, not adding a new uniform

Comment (thevandie) [reply ModularBungalow/clothing/neocoat.dm:86]: But people want it to be a new uniform. So you should.

**Comment by thevandie** [comment]
Also, fyi, community vote after the weekend. Stuffs gonna need to be changed if it's unpopular.

File: ModularBungalow/clothing/neocoat.dm
Comment (Kitsunemitsu) [reply ModularBungalow/clothing/neocoat.dm:86]: > Its resprite, not adding a new uniform

If the community says no, That means no. Going around to make another PR to try and circumvent proper opinion is disrespectful. 
It doesn't matter what it was intended as, 
**This server is a democracy.**

**Comment by Kitsunemitsu** [comment]
Quit Bickering, Wait for the feedback vote. Further comments on the matter will be removed if offtopic

**Comment by thevandie** [comment]
Issue, galaxy cap's carapace is an error for item sprite.

**Comment by retlaw34** [comment]
Keep this TMed for the weekend, ill try to atomize this with what was voted yes

**Pull request closed without merging**
